app/helpers/application_helper.rb

```diff
@@ -25,7 +25,8 @@ def display_time_slot(times, glue=' to ')
 
   def display_slot_and_duration(times, glue=', ')
     from, to = times.split('-')
-    [format_time_str(from).strip, (time_from_str(to)-time_from_str(from)).duration].join(glue)
+    [format_time_str(from).strip, (time_from_str(to)-time_from_str(from)).
+      duration].join(glue)
```

Comment:
This might be clearer formatted as:

```ruby
[
  format_time_str(from).strip,
  (time_from_str(to)-time_from_str(from)).duration
].join(glue)
```